### ISSUE TITLE
🙋‍♀️ 1-click survey: theme mods

### ISSUE THREAD

**Issue by squidfunk** [squidfunk/mkdocs-material#2071]
I'm thinking about making a little series of mods for Material for MkDocs, showing how the look of the theme can be customized with custom CSS in a way that it doesn't look like common Material Design. Vote up or down:

👍 – Great idea, I'd like to learn more about this
❤️ – I'm happy with how Material for MkDocs looks
👎 – No thanks, I can do that myself

Mods would be available as extra CSS which can be dropped into `mkdocs.yml`.

An example for Admonitions:

<img width="741" alt="Bildschirmfoto 2020-11-29 um 15 07 03" src="https://user-images.githubusercontent.com/932156/100545656-2b049500-325d-11eb-84e0-68e322294e10.png">

**Comment by squidfunk** [comment]
The admonition mod has been released as part of Insiders!

1. Copy & paste the [following CSS](https://github.com/squidfunk/mkdocs-material-insiders/blob/master/examples/modifications/admonition.css)
2. Add it as an [additional stylesheet](https://squidfunk.github.io/mkdocs-material/customization/#additional-css)
3. Voila!

<details><summary>Light preview</summary>
<img src="https://user-images.githubusercontent.com/932156/100654868-bc9e0080-334a-11eb-9fff-bc5c4e05164e.png"/>
</details>

<details><summary>Slate preview</summary>
<img src="https://user-images.githubusercontent.com/932156/100655069-0edf2180-334b-11eb-8df3-6325df2128bb.png" />
</details>

**Comment by Paouky** [comment]
Oh, this is really great! I'd definitely love to use it for https://docs.grin.mw/

**Comment by RDIL** [comment]
> The admonition mod has been released as part of Insiders!
> 
> 1. Copy & paste the [following CSS](https://github.com/squidfunk/mkdocs-material-insiders/blob/master/examples/modifications/admonition.css)
> 2. Add it as an [additional stylesheet](https://squidfunk.github.io/mkdocs-material/customization/#additional-css)
> 3. Voila!
> 
> Light preview
> Slate preview

Clarification question: since it is part of insiders, does that mean it isn't usable because including it in public repositories would be against the policy for insiders which says it can't be publicly redistributed?  Would I need to download it during build time, or just include it in the repository?

**Comment by squidfunk** [comment]
__TL;DR: Of course, you can use it in all your public repositories without any restrictions.__

Thanks for asking! The source of Insiders is still released under MIT – the source is just not accessible to non-sponsors. The terms for Insiders just propose a _fair-use policy_, i.e. that users don't release the source code (i.e. make the repository accessible), as that would cannibalize the whole strategy and harm the community. In theory, the sources can still be downloaded and deconstructed from the official Insiders-powered documentation, but that imposes a significant degree of work, plus it's non-sustainable regarding changes, i.e. you'd have to do it on new version.

I understand, that there's a "threat" of people "stealing" the snippets when they see them somewhere, but I'm fine with that if somebody wants to go that extra mile. The snippet can just be dropped in as an additional stylesheet, see the linked docs. I deliberately decided against integrating them as part of the build process, as I want to make them instantly tweakable, so you can change or throw out the stuff you don't like.

**Comment by squidfunk** [comment]
Closing. Thanks to all voters! I'll add more theme mods to Insiders within the coming weeks!

**Comment by fveauvy** [comment]
@squidfunk : the slate version seems a bit odd when the admonition is closed.

<img width="187" alt="Capture d’écran 2020-12-17 à 21 48 37" src="https://user-images.githubusercontent.com/38651917/102542432-1411cf80-40b2-11eb-9b0d-be83537519da.png">

I added the following `background-color` override for the closed state style and it's working for both themes versions :

<img width="1057" alt="Capture d’écran 2020-12-17 à 21 50 02" src="https://user-images.githubusercontent.com/38651917/102542461-1e33ce00-40b2-11eb-977e-bf47bd8959bf.png">

Result: 

<img width="194" alt="Capture d’écran 2020-12-17 à 21 50 52" src="https://user-images.githubusercontent.com/38651917/102542626-5f2be280-40b2-11eb-89ac-919091e3f716.png">

**Comment by squidfunk** [comment]
That's awesome, thanks for taking the time to fix it! Could you create a PR in the Insiders repo?

**Comment by fveauvy** [comment]
I created a PR following the contributing guidance.
